processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -453,7 +453,10 @@ TimeAndDims toTimeAndDims(InputRow row) throws IndexSizeExceededException
       dims = newDims;
     }
 
-    long truncated = gran.truncate(row.getTimestampFromEpoch());
+    long truncated = 0;
+    if (row.getTimestamp() != null) {
```

Comment:
dnt think this should be null for any valid case, is there any case when it will be null ?